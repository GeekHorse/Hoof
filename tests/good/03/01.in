most right
new right four done
most left
new left three done
most right
new right five done
most left
new left two done
most right
new right six done
most left
new left one done
save
