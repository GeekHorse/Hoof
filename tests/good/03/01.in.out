new right one two three four five six done
